### DISCUSSION TITLE
change the port

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [FlowiseAI/Flowise#207] [Q&A] [answered]
most of the time port 3000 is in use, how can I change it for Flowise? `flowise start -p=3001` doesn't work!

**Comment by HenryHengZJ** [comment]
thanks for the bug! will make the changes to allow port change on npm installation

**Reply by HenryHengZJ** [reply, marked as answer]
should now be able to use `npx flowise start --PORT=5000`
